test/test_core.py

```diff
@@ -4390,21 +4391,6 @@ def test_dylink_static_funcpointer_float(self):
       expected='hello 1: 56.779999\ngot: 1\nhello 1: 12.340000\n',
       header='typedef float (*floatfunc)(float);', force_c=True)
 
-  @needs_dylink
-  def test_missing_signatures(self):
-    create_file('test_sig.c', r'''#include <emscripten.h>
-                                       int main() {
-                                         return 0 == ( (long)&emscripten_run_script_string +
-                                                       (long)&emscripten_run_script );
```

Comment:
Is this signature part of the test not needed?

Reply:
This test was added in #13355 but it is badly names and is trying to test the validity of the `__sig` for `emscripten_run_script_string`.. but these days the sigs are automatically generated and we don't need such individual tests like this.